router/worker.go

```diff
@@ -166,6 +166,7 @@ func (w *worker) workLoop() {
 				UserID:             userID,
 				JobID:              job.JobID,
 				SourceID:           parameters.SourceID,
+				SourceCategory:     parameters.SourceCategory,
```

Comment:
We are using [similar logic](https://github.com/rudderlabs/rudder-server/blob/master/router/handle.go#L354) to populate sourceCategory for reporting